### PR TITLE
adding reset/change password

### PR DESCRIPTION
**Pull request by noterudite**

### CODE REVIEW

File: afm-backend/controller/userController.py [hunk 1]
```diff
@@ -22,4 +23,25 @@ def login():
     db.commit()    
     access_token = create_access_token(identity=username, expires_delta=datetime.timedelta(minutes=60))
     db = None
-    return jsonify(access_token=access_token)
\ No newline at end of file
+    return jsonify(access_token=access_token)
+
+@user_blueprint.route("/reset_password", methods=["PUT"])
```
Comment (bhavan0) [inline afm-backend/controller/userController.py:28]: change _ to -

File: afm-backend/controller/userController.py [hunk 2]
```diff
@@ -22,4 +23,25 @@ def login():
...
+@jwt_required()
+def reset_password():
+    db = Database()
```
Comment (bhavan0) [inline afm-backend/controller/userController.py:31]: Move all the db logic to userRepository

Then add the method in userDomain

And then call that from here

File: afm-backend/controller/userController.py [hunk 3]
```diff
@@ -22,4 +23,25 @@ def login():
...
+    cursor = db.cursor()
+    username = get_jwt_identity()
+    password = request.json.get("password", None)
+    confirm_password = request.json.get("confirm_password", None)
```
Comment (bhavan0) [inline afm-backend/controller/userController.py:35]: Change confirm_password to confirmPassword
To keep it consistent with other APIs

File: afm-backend/controller/userController.py (hunk 1 above)
Comment (noterudite) [reply afm-backend/controller/userController.py:28]: Got it

File: afm-backend/controller/userController.py (hunk 2 above)
Comment (noterudite) [reply afm-backend/controller/userController.py:31]: Cool

File: afm-backend/controller/userController.py (hunk 3 above)
Comment (noterudite) [reply afm-backend/controller/userController.py:35]: Got it

**Pull request closed without merging**